ios/Classes/MapEvents+JSON.swift

```diff
@@ -9,8 +9,8 @@ protocol MapEventEncodable {
 extension EventTimeInterval {
     var toJSON: [String: Any] {
         [
-            "begin": begin,
-            "end": end
+            "begin": begin.timeIntervalSince1970,
```

Comment:
I think on the flutter side this is represented in microseconds, but his is in seconds.

Reply:
ah I just know it, will fix it soon